C++ Compiler Warning

Please fix 
```
AstTranslator.cpp:1337:9: warning: explicitly defaulted default constructor is implicitly deleted
      [-Wdefaulted-function-deleted]
        VariablesToArguments() = default;
        ^
AstTranslator.cpp:1335:48: note: default constructor of 'VariablesToArguments' is implicitly
      deleted because field 'uniqueVariables' of reference type 'const std::vector<const
      AstVariable *> &' would not be initialized
        const std::vector<const AstVariable*>& uniqueVariables;
```

--- a/src/AstTranslator.cpp
+++ b/src/AstTranslator.cpp
@@ -1335,7 +1335,6 @@ std::unique_ptr<RamStatement> AstTranslator::makeNegationSubproofSubroutine(cons
     struct VariablesToArguments : public AstNodeMapper {
         const std::vector<const AstVariable*>& uniqueVariables;
 
-        VariablesToArguments() = default;
         VariablesToArguments(const std::vector<const AstVariable*>& uniqueVariables)
                 : uniqueVariables(uniqueVariables) {}
 

